packages/server/graphql/mutations/inviteToTeam.ts

```diff
@@ -140,8 +134,8 @@ export default {
             inviteLink: makeAppURL(appOrigin, `team-invitation/${invitation.token}`, options),
             inviteeName: user ? user.preferredName : '',
             inviteeEmail: invitation.email,
-            inviterName: inviter.preferredName,
-            inviterEmail: inviter.email,
+            inviterName: inviter!.preferredName,
```

Comment:
I would use some check like `if (!inviter) throw error`, if not obvious directly from code above that `null` is impossible. What do you think?

Reply:
done!